modules.d/95iscsi/module-setup.sh

```diff
@@ -198,6 +198,9 @@ install() {
         "$systemdsystemunitdir"/iscsiuio.socket \
         "$systemdsystemunitdir"/sockets.target.wants/iscsid.socket \
         "$systemdsystemunitdir"/sockets.target.wants/iscsiuio.socket
+    if grep -q '^ExecStartPre=/usr/lib/open-iscsi/startup-checks.sh$' "$systemdsystemunitdir/iscsid.service"; then
+        inst_simple /usr/lib/open-iscsi/startup-checks.sh
```

Comment:
It might be sufficient to just use `inst_simple -o` (`-o`: "If SOURCE does not exist, do not fail").

Reply:
The current approach is more strict. It will fail in case /usr/lib/open-iscsi/startup-checks.sh was found in the service file but fails to install.

Reply:
ok, fine with me.